interface/modules/zend_modules/public/js/autosuggest/autosuggest.js

```diff
@@ -19,96 +19,99 @@
-        searchMode  : searchMode,
-      }, function(data){
-				cache: false;
-				if(data.length > 0) {
-          if (searchEleNo > 0) {
-            var inputStringValue = $('#inputString' + searchEleNo).val();
-          } else {
-            var inputStringValue = $('#inputString').val();
-          }
-					if (inputStringCheck == inputStringValue) {
-						if (searchEleNo > 0) {
-              $('#suggestions' + searchEleNo).show();
-              $('#autoSuggestionsList' + searchEleNo).html(data);
-            } else {
-              $('#suggestions').show();
-              $('#autoSuggestionsList').html(data);
-            }
-            // Focus to 1st Row
-            if(document.getElementById('list_' + searchEleNo + '_1')) {
-                $('#list_' + searchEleNo + '_1').focus();
-                $('#list_' + searchEleNo + '_1').css("background-color","#659CD8");
-            }
-					}
-				}
-			});
-		}
-	}
+        const leading = '%';
+        const trailing = '%';
+        const page = $('#page').val();
+        /** Path settings */
+        const path = window.location;
+        const arr = path.toString().split('public');
+        const count = arr[1].split('/').length - 1;
+        let newPath = './';
+        for (let i = 0; i < count; i += 1) {
+            newPath += '../';
+        }
+
+        // dataType: "html",
+        //     $.post(newPath + "public/application/index/search", {
+        //         queryString: inputString,
+        //         leading: leading,
+        //         trailing: trailing,
+        //         page: page,
+        //         searchType: searchType,
+        //         searchEleNo: searchEleNo,
+        //         searchMode: searchMode,
+        //     }, function (data) {
+        //         cache: false;
+        //         if (data.length > 0) {
+        //             if (searchEleNo > 0) {
+        //                 var inputStringValue = $('#inputString' + searchEleNo).val();
+        //             } else {
+        //                 var inputStringValue = $('#inputString').val();
+        //             }
+        //             if (inputStringCheck == inputStringValue) {
+        //                 if (searchEleNo > 0) {
+        //                     $('#suggestions' + searchEleNo).show();
+        //                     $('#autoSuggestionsList' + searchEleNo).html(data);
+        //                 } else {
+        //                     $('#suggestions').show();
+        //                     $('#autoSuggestionsList').html(data);
+        //                 }
+        //                 // Focus to 1st Row
+        //                 if (document.getElementById('list_' + searchEleNo + '_1')) {
+        //                     $('#list_' + searchEleNo + '_1').focus();
+        //                     $('#list_' + searchEleNo + '_1').css("background-color", "#659CD8");
+        //                 }
+        //             }
+        //         }
+        //     });
```

Comment:
Hi  @bradymiller 

Not sure about this code, guess they are unnecessary or brokered,
so I commented out the code need you to confirm.